src/transforms/transform-code-scanning-alert.ts

```diff
@@ -0,0 +1,93 @@
+import { jiraIssueKeyParser } from "utils/jira-utils";
+import {Context} from "probot/lib/context";
+import {JiraRemoteLinkData, JiraRemoteLinkStatusAppearance} from "interfaces/jira";
+
+const MAX_STRING_LENGTH = 255;
+
+const getPullRequestTitle = async (repoName: string, prId: number, repoOwner: string, context: Context): Promise<string> => {
+	const response = await context.github.pulls.get({
+		owner: repoOwner,
+		repo: repoName,
+		pull_number: prId
+	});
+
+	if (response.status !== 200) {
+		context.log.warn(`Received error when querying for Pull Request information: ${JSON.stringify(response.data)}`);
```

Comment:
don't stringify the data in the message, just add it as data to the log `context.log.warn({response}, "Received error when...")`